### ISSUE TITLE
Why is `context` undefined for jest / jasmine interfaces?

### ISSUE THREAD

**Issue by pmoons** [stalniy/bdd-lazy-var#54]
[Jest/Jasmine Interfaces
](https://github.com/stalniy/bdd-lazy-var/blob/master/lib/interface/jasmine.js#L24-L32)
Yet defined for the [mocha interface](https://github.com/stalniy/bdd-lazy-var/blob/master/lib/interface/mocha.js#L31-L41)?

**Comment by pmoons** [comment]
So because `jest` and `jasmine` don't have interfaces that can provide context variables to plugins, we can't use `context` when testing with these testing libraries?

It would be nice to be able to do the following with `jest` or `jasmine`:

```javascript
describe("A thing", () =>
  it("does something right", () => { ... }

  context("when something goes wrong", () => {
    it("does something wrong", () => { ... }
});
```

**Comment by stalniy** [comment]
Ah, now I see what you mean :) you want `context` to be an alias of `describe` when using jest or jasmine

**Comment by stalniy** [comment]
The answer to the question because standard BDD interface of mocha provides `context` variable but jasmine and jest no, so I’ve not thought that this may be important for somebody to have `context` keyword in other frameworks

**Comment by pmoons** [comment]
I really enjoy the semantics of `rspec`, especially how `context` breaks up the edge cases from the "happy path" case presented in the top level `describe` block.  Since jasmine and jest just re-use the same word to present these edge cases, would it hurt to have the word `context` be aliased to `describe`?

**Comment by stalniy** [comment]
I think that's fine. Could you please make a PR? You need to update this file https://github.com/stalniy/bdd-lazy-var/blob/master/lib/interface/jasmine.js

Please don't forget to add `xcontext` and `fcontext` aliases for `xdescribe` & `fdescribe`

**Comment by stalniy** [comment]
released in `bdd-lazy-var@2.5.0`, thanks for the help!
